src/stores/modelStore.ts

```diff
@@ -48,6 +49,19 @@ export class ComfyModelDef {
   is_load_requested: boolean = false
   /** If true, this is a fake model object used as a placeholder for something (eg a loading icon) */
   is_fake_object: boolean = false
+  /** A string full of auto-computed lowercase-only searchable text for this model */
```

Comment:
You should not use `ref` or `computed` in a plain object. They should only be used in the context of Pinia store and Vue component. The only exception now in this repo is the nodeBadge extension, which will be migrated to be a vue component later.

The searchable here can just be a getter method. It will just be reactive automatically.

Reply:
I don't need a reactive value, I need a cached value.
This also *is* in the context of a pinia store.

Reply:
`computed` should only be used at top level of pinia store declaration (compositional API).

Reply:
Marking the whole object raw will make the all rest of attributes non-reactive.

Reply:
How can I achieve a caching value in this context

Reply:
(noting for the record that the current code does seem to work exactly as intended)

Reply:
(actually I could be wrong, idk if the cache resets as intended, as that's the part it tries to use reactivity for)

Should I just store a string and update it manually instead?

Reply:
eh, why not, applied - it's just a simple string now, with a function to update it when needed